ceph-releases/ALL/ubi9-ibm/daemon-base/__CEPH_IBM_PACKAGES__

```diff
@@ -0,0 +1 @@
+ceph-call-home
```

Comment:
@guits @ktdreyer is this the package name for `yum install`?
If so, I believe it should be `ceph-mgr-callhome`

Reply:
You're right, it's `ceph-mgr-callhome`
```suggestion
ceph-mgr-callhome
```